YourJourney/Assets/Scripts/Engine/Managers/TriggerManager.cs

```diff
@@ -199,7 +199,7 @@ IEnumerator TriggerChain()
 			}
 			yield return WaitUntilFinished();
 
-			if ( trigger.dataName == endTriggerGUID )
+			if ( trigger.dataName == endTriggerGUID && endTriggerGUID?.Length > 0)
```

Comment:
Approved. But do you think we should use one of these to make it more clear?
!string.IsNullOrEmpty(endTriggerGUID)
!string.IsNullOrWhiteSpace(endTriggerGUID)

Reply:
Not sure which one is actually clearer, but changed this to IsNullOrEmpty